starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -75,4 +75,34 @@ public void create() {
     // the IDs should NOT match because the ID gets set to null in the create method
     Assert.assertNotEquals("FIRST", this.mongoTemplate.findAll(Inventory.class).get(0).getId());
   }
+
+  /**
+   * Test Delete method.
+   */
+  @Test
+  public void delete() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    inventory.setDescription("Product 1");
+    this.inventoryDAO.create(inventory);
+    this.inventoryDAO.create(inventory);
+
+    List<Inventory> inventoryList = this.mongoTemplate.findAll(Inventory.class);
+    Assert.assertEquals(2, inventoryList.size());
+
+    //deletes the second inventory Object
+    this.inventoryDAO.delete(inventoryList.get(1).getId());
```

Comment:
If the create method sets the Id to null, how wouldn't the result of .getId() be null for both inventory objects? Do you need to set the inventoryId in the two test inventory objects you made in this test?